scripts/build_loop.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -eu
+
+SCRIPT="$(basename "${0}")"
+BUILDS_DIR=build
+LOOPWORKSPACE_DIR=LoopWorkspace
+
+error() {
+  echo "ERROR: ${*}" >&2
+  echo "Usage: ${SCRIPT} <build_tag>" >&2
+  echo "Parameters:" >&2
+  echo "  <build_tag>	tag of the version to build, or if not found it will be cloned e.g. `build-277`;" >&2
+  exit 1
+}
+
+info() {
+  echo "INFO: ${*}" >&2
+}
+
+if [ ${#} -lt 1 ]; then
+  error "Missing arguments"
+fi
+
+BUILD_TAG="${1}"
+shift 1
+
+if [ ${#} -ne 0 ]; then
+  error "Unexpected arguments: ${*}"
+fi
+
+if [ ! -d "${BUILDS_DIR}" ]; then
+  mkdir ${BUILDS_DIR}
+fi
+
+cd ${BUILDS_DIR}
```

Comment:
[nit] I believe I had suggested using a different directory for each build tag. So, `builds/build-123`, rather than a shared `builds/LoopWorkspace` directory that just switches code. However, I'm fine either way as long as switching tags in the same repo doesn't have unintended side effects (from pre-build Carthage directories, Xcode local files, etc.)